src/api/app/controllers/webui/package_controller.rb

```diff
@@ -92,22 +92,25 @@ def linking_packages
   end
 
   def dependency
-    unless Project.find_by_name(params[:dproject].to_s)
+    dep_project = Project.find_by_name(params[:dproject]) || Project.find_remote_project(params[:dproject]).try(:first)
```

Comment:
:thought_balloon: `dep_project` `:dproject`, please use speaking variable names.

Reply:
You know one is an object and one is a string right? Not sure if it's `nil` or `''`....

Reply:
`Project.find_remote_project(...)` returns an array with the first element being a `Project` object. [See this line](https://github.com/openSUSE/open-build-service/blob/cf5da985e2a20f00af70b4e4a4b41aa1b3b3e131/src/api/app/models/project.rb#L484)

Reply:
Ah true that's `get_by_name`. Lovely...